services-custom/dynamodb-enhanced/README.md

```diff
@@ -134,7 +156,7 @@ key differences:
    an asynchronous DynamoDb client from the SDK as well):
    ```java
     AsyncMappedDatabase database = DynamoDbAsyncMappedDatabase.builder()
-                                                              .dynamoDbAsyncClient(dynamoDbAsyncClient)
+                                                              .dynamoDbClient(dynamoDbAsyncClient)
```

Comment:
Not sure if you also want to update the name of the arg

Reply:
That was deliberate to show that we were actually passing in an async client.